thundra/opentracing/tracer.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, recorder=None, scope_manager=None):
         self.global_span_order = 0
         self.test_xray_traces = []
         ThundraTracer.__instance = self
+        self.function_name = None
```

Comment:
It is better we should not have something specific to function here. I don't know the requirement of having `function_name` here but it should not be handled through tracer

Reply:
I mimicked this from NodeJS agent. It is used to carry function name to integrations.